src/main/java/tech/artisanhub/ShapeletTrainer/LearnShapelets.java

```diff
@@ -18,14 +18,14 @@ public static void main( String[] args )
 
             int minLength = 1;
             int maxLength = 4;
-            String salihPath = "home/jawadhsr/Desktop/FYP/FIles/iris_replacedNamedWithInts.data";
+            String salihPath = "/home/jawadhsr/Desktop/FYP/FIles/iris_replacedNamedWithInts.arff";
             String ARFFName = "/home/rnavagamuwa/Documents/CSE/FYP/Datasets/pima-indians-diabetes/arff-pima-indians-diabetes.data";
-            Instances data = ShapeletFilter.loadData(ARFFName);
+            Instances data = ShapeletFilter.loadData(salihPath);
 
-            String salihOuputPath =  "home/jawadhsr/Desktop/FYP/FIles/Shaplets.txt";
+            String salihOuputPath =  "/home/jawadhsr/Desktop/FYP/FIles/Shaplets.txt";
             String outPutFile = "/home/rnavagamuwa/Documents/CSE/FYP/Datasets/pima-indians-diabetes/shapeletsOut.txt";
             ShapeletFilter sf = new ShapeletFilter(k, minLength, maxLength);
-            sf.setLogOutputFile(outPutFile); // log file stores shapelet output
+            sf.setLogOutputFile(salihOuputPath); // log file stores shapelet output
             ArrayList<Shapelet> generatedShapelets = sf.process(data);
         }
         catch (Exception e) {
```

Comment:
@Jawadh-Salih machan remove this file from the commit. This is not actually a change of code. Will be very hard to track if something goes wrong with these kind of commits :smile: